keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -100,6 +100,9 @@ func addJoinOtherFlags(cmd *cobra.Command, initOpts *types.InitOptions) {
 
 	cmd.Flags().StringVar(&initOpts.DNS, types.DomainName, initOpts.DNS,
 		"Use this key to set domain names in cloudcore's certificate SubAltNames field. eg: www.cloudcore.cn,www.kubeedge.cn")
+
+	cmd.Flags().StringVar(&initOpts.TarballPath, types.TarballPath, initOpts.TarballPath,
+		"Use this key to set the temp directory path for downloading kubeedge installation package")
```

Comment:
Better as "Use this key to set the temp directory path for KubeEdge tarball, if not exist, download it"